rfcs/0052-create-sink-into-table.md

```diff
@@ -0,0 +1,41 @@
+---
+feature: create_sink_into_table
+authors:
+  - "st1page"
+start_date: "2023/02/15"
+---
+
+# Create Sink into Table
+
+## Motivation
+
+### Loss of functionality compared to FlinkSQL
+FlinkSQL uses the `INSERT ... SELECT ...` statement to create a streaming job. This means that in FlinkSQL, the table content is not bound to the result of a streaming query, even if the table is stored in Flink table store (https://nightlies.apache.org/flink/flink-table-store-docs-master/). **Users can sink more than one stream into a table and let the table handle the pk conflict by itself**.
+
+However, RisingWave uses `create materialised view as query' to declare a streaming query and bind a table with the query's result set. So compared to FlinkSQL, RisingWave cannot express multiple streaming query into one internal table naturally.
```

Comment:
Honestly speaking, I don't think this is a good practice, e.g. multi insert statement into one table. Why not just use union/join in one select query? Also, this may introduce many problems such as concurrent modification error in lake house format.